package.json

```diff
@@ -10,9 +10,11 @@
   },
   "dependencies": {
     "@reduxjs/toolkit": "^1.9.1",
+    "antd": "^5.1.7",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-redux": "^8.0.5"
+    "react-redux": "^8.0.5",
+    "react-router-dom": "^6.8.0"
   },
   "devDependencies": {
     "@types/react": "^18.0.26",
```

Comment:
We already have a yarn.lock file. We have no need for this.
Also, it is a bad practice to keep 2 lockfiles.

So, please delete this one and install dependencies using yarn.